packages/jsii-pacmak/lib/targets/python.ts

```diff
@@ -1642,6 +1694,12 @@ class Package {
       code.openFile(filename);
       mod.emit(code, context);
       code.closeFile(filename);
+
+      for (const script of mod.emitBinScripts(code)) {
+        if (scripts.indexOf(script) < 0) {
+          scripts.push(script);
+        }
```

Comment:
I think the duplication defense here is not necessary (it also does not hurt!)

Reply:
It does not hurt but looks strange when the same script appeared several times.

Reply:
But... it shouldn't... I mean a script can only be registered once... so if we have duplicates, it could mean something wrong happened somewhere else?

Reply:
You are right. I have made some tests and have not found any dublicates.